File: project.cs
using System;

namespace UcakRezervasyonProgrami
{
    internal class Program
    {
        public static string IsimSoyisim;

        static void Main(string[] args)
        {
            DateTime kalkisZamani1 = new DateTime(2023, 10, 05, 10, 30, 0);
            DateTime kalkisZamani2 = new DateTime(2023, 10, 12, 10, 30, 0);
            DateTime kalkisZamani3 = new DateTime(2023, 10, 17, 10, 30, 0);
            DateTime kalkisZamani4 = new DateTime(2023, 10, 24, 10, 30, 0);

            UcakBilgileri ucak = new UcakBilgileri();
            ucak.model = "Kargo Uçağı";
            ucak.seriNo = 20204382691347;
            ucak.yolcuKapasitesi = 50;
            ucak.tasimaKapasitesi = 200000;
            ucak.bakimTarihi = "01, 10, 2023";
            ucak.ucakRengi = "koyu yeşil";

            KargoBilgileri kargo = new KargoBilgileri();
            Console.WriteLine("Gönderenin İsimi ve Soyadı:");
            kargo.gonderenAdSoyad = Console.ReadLine();
            Console.WriteLine("Kargonun Türünü Giriniz: (Kırılacak, Boya, Ürün vb.)");
            kargo.tur = Console.ReadLine();
            Console.WriteLine("Kilogram Cinsinden Ağırlığını Giriniz:");
            kargo.agirlik_kg = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Gönderenin Numarasını Giriniz (XXXX) :");
            kargo.gondericiNo = Convert.ToInt64(Console.ReadLine());
            Console.WriteLine("Alıcının Numarasını Giriniz (XXXX) :");
            kargo.aliciNo = Convert.ToInt64(Console.ReadLine());
            Console.WriteLine("İletişim Bilgilerinizi Giriniz (Telefon Numarası) +90");
            kargo.iletisimBilgileri = Console.ReadLine();

            LokasyonBilgileri lokasyon = new LokasyonBilgileri();
            Console.WriteLine("Kargonun Gideceği Ülkeyi Giriniz:");
            lokasyon.ulke = Console.ReadLine();
            Console.WriteLine("Kargonun Gideceği Şehri Giriniz:");
            lokasyon.sehir = Console.ReadLine();
            Console.WriteLine("Kargonun Gönderileceği Havaalanını Giriniz:");
            lokasyon.havaalani = Console.ReadLine();
            Console.WriteLine("Doluluk Nedeniyle En Yakın Tarih Olan " + kalkisZamani1 + " Tarihine " + lokasyon.ulke + "/" + lokasyon.sehir + "/" + lokasyon.havaalani + " Konumunda Kaydınız Oluşturulmuştur.");
        }
    }

    internal class UcakBilgileri
    {
        public string model;
        public long seriNo;
        public int yolcuKapasitesi;
        public int tasimaKapasitesi;
        public string bakimTarihi;
        public string ucakRengi;
    }

    internal class KargoBilgileri
    {
        public string gonderenAdSoyad;
        public string tur;
        public int agirlik_kg;
        public long gondericiNo;
        public long aliciNo;
        public string iletisimBilgileri;
    }

    internal class LokasyonBilgileri
    {
        public string ulke;
        public string sehir;
        public string havaalani;
    }
}

File: project.java
import java.util.Scanner;

public class UcakRezervasyonProgrami {
    public static String IsimSoyisim;

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String kalkisZamani1 = "2023-10-05T10:30:00";
        String kalkisZamani2 = "2023-10-12T10:30:00";
        String kalkisZamani3 = "2023-10-17T10:30:00";
        String kalkisZamani4 = "2023-10-24T10:30:00";

        UcakBilgileri ucak = new UcakBilgileri();
        ucak.model = "Kargo Uçağı";
        ucak.seriNo = 20204382691347L;
        ucak.yolcuKapasitesi = 50;
        ucak.tasimaKapasitesi = 200000;
        ucak.bakimTarihi = "01, 10, 2023";
        ucak.ucakRengi = "koyu yeşil";

        KargoBilgileri kargo = new KargoBilgileri();
        System.out.println("Gönderenin İsimi ve Soyadı:");
        kargo.gonderenAdSoyad = scanner.nextLine();
        System.out.println("Kargonun Türünü Giriniz: (Kırılacak, Boya, Ürün vb.)");
        kargo.tur = scanner.nextLine();
        System.out.println("Kilogram Cinsinden Ağırlığını Giriniz:");
        kargo.agirlik_kg = Integer.parseInt(scanner.nextLine());
        System.out.println("Gönderenin Numarasını Giriniz (XXXX) :");
        kargo.gondericiNo = Long.parseLong(scanner.nextLine());
        System.out.println("Alıcının Numarasını Giriniz (XXXX) :");
        kargo.aliciNo = Long.parseLong(scanner.nextLine());
        System.out.println("İletişim Bilgilerinizi Giriniz (Telefon Numarası) +90");
        kargo.iletisimBilgileri = scanner.nextLine();

        LokasyonBilgileri lokasyon = new LokasyonBilgileri();
        System.out.println("Kargonun Gideceği Ülkeyi Giriniz:");
        lokasyon.ulke = scanner.nextLine();
        System.out.println("Kargonun Gideceği Şehri Giriniz:");
        lokasyon.sehir = scanner.nextLine();
        System.out.println("Kargonun Gönderileceği Havaalanını Giriniz:");
        lokasyon.havaalani = scanner.nextLine();
        System.out.println("Doluluk Nedeniyle En Yakın Tarih Olan " + kalkisZamani1 + " Tarihine " + lokasyon.ulke + "/" + lokasyon.sehir + "/" + lokasyon.havaalani + " Konumunda Kaydınız Oluşturulmuştur.");
    }
}

class UcakBilgileri {
    public String model;
    public long seriNo;
    public int yolcuKapasitesi;
    public int tasimaKapasitesi;
    public String bakimTarihi;
    public String ucakRengi;
}

class KargoBilgileri {
    public String gonderenAdSoyad;
    public String tur;
    public int agirlik_kg;
    public long gondericiNo;
    public long aliciNo;
    public String iletisimBilgileri;
}

class LokasyonBilgileri {
    public String ulke;
    public String sehir;
    public String havaalani;
}
